features/src/main/feature/spring-features.xml

```diff
@@ -38,6 +38,13 @@
         <bundle start-level="30">mvn:org.apache.servicemix.bundles/org.apache.servicemix.bundles.spring-tx/${spring.version}_1</bundle>
     </feature>
 
+    <feature name="spring-orm" version="${spring.version}">
+        <feature version='${spring.version}'>spring-tx</feature>
+        <feature version='${spring.version}'>spring-jdbc</feature>
+        <bundle dependency='true'>mvn:org.apache.servicemix.bundles/org.apache.servicemix.bundles.aopalliance/1.0_6</bundle>
```

Comment:
https://central.sonatype.com/artifact/org.apache.servicemix.bundles/org.apache.servicemix.bundles.spring-orm/6.0.12_1
- `Import-Package` contains `org.aopalliance.intercept;version="[1,2)"`, although this package is present inside `spring-aop` jar, it isn't exported by the bundle